### DISCUSSION TITLE
Support .NET 6+

### DISCUSSION THREAD

**Discussion by erikbartlow** [loic-sharma/BaGet#750] [Q&A]
My packages are currently configured to use .NET 6 and targeting Windows OS.  When I try and view the packages in the package manager I get a 404.  I assume it is due to the .NET 6 framework since I don't see that option in the search criteria on the packages page.   How do I configure BaGet to support .net 6+ and any OS type? 

&targetFramework='net6.0-windows7.0

**Comment by NilkOne** [comment]
Hi
same problem, here 
@erikbartlow: did you find a solution?
